### ISSUE TITLE
basicAuth Configuration Not Working Properly for Multiple Routes

### ISSUE THREAD

**Issue by rathahin** [Baroshem/nuxt-security#266]
## Description

I'm experiencing an issue where the `basicAuth` configuration doesn't work as expected for multiple routes. While it works perfectly for a single route, it fails to operate correctly when more than one route is specified.

### Steps to Reproduce

1. Set the `basicAuth` configuration to include multiple paths like `[ '/e-voucher-uat', '/en/e-voucher-uat', '/ku/e-voucher-uat' ]`.
2. Try to access any of the paths.

### Expected Behavior

I expect that accessing any of the paths specified in the configuration would require basic authentication.

### Actual Behavior

Only the first path in the configuration array (`/e-voucher-uat` in this case) is requiring basic authentication. Other paths are not prompting for authentication.

### Code Snippet (Optional)

Upon checking the `basicAuth.mjs` file, I noticed that the `.some()` method is used to check the paths. Here's the relevant code snippet:

```javascript
basicAuthConfig?.include?.some((el) => !event.path?.startsWith(el))
```
The `.some()` method stops iterating as soon as it finds a match, causing only the first path to work as expected.

### Environment

- Nuxt version: 3.5.0
- nuxt-security: 1.0.0-rc.2

### Additional Information

None

### Possible Solution

Consider using `.forEach()` or another looping mechanism that will check all paths before deciding to skip authentication.

**Comment by Baroshem** [comment]
Hey @rathahin 

Thanks for this issue! The `.some()` method is not good here, you are correct. 

Would you be maybe interested in creating a Pull Request with your change? 

I can provide all help needed :)

**Comment by rathahin** [comment]
Okay, let me try.

**Comment by Baroshem** [comment]
Closing as it was released with 1.0.0-rc.3 :)
